packages/rollup-dev-tools/test/transpiler/transpile.jump.spec.ts

```diff
@@ -149,7 +152,10 @@ describe('Transpile - JUMPs', () => {
 
   it('handles simple JUMPs properly', async () => {
     const evmBytecode: EVMBytecode = [
-      { opcode: Opcode.PUSH32, consumedBytes: bufferUtils.numberToBuffer(34) },
+      {
+        opcode: Opcode.PUSH1,
+        consumedBytes: bufferUtils.numberToBufferPacked(3, 2),
```

Comment:
Maybe I'm missing something, but shouldn't this break since it's a `PUSH1` but consumedBytes has `minLength = 2`?